Sources/SIL/SILPrinter.swift

```diff
@@ -157,12 +164,25 @@ class SILPrinter: Printer {
             print(type)
             print(", ")
             literal(value)
-        case let .load(operand):
+        case let .load(maybeOwnership, operand):
             print("load ")
+            if let ownership = maybeOwnership {
+                print(ownership)
+                print(" ")
+            }
```

Comment:
`print(maybeOwnership, " ")`?

Reply:
Ok, if we really want to code golf this, then I just realized that the current pattern for handling cases like this is
```
print(instructionDef.result, " = ") { print($0) }
```
which seems _really_ awkward to me. I can change it to that if you feel strongly about it though.

Reply:
We also use the `if` pattern [here (twice)](https://github.com/tensorflow/swift/blob/master/Sources/SIL/SILPrinter.swift#L322-L330) and [here](https://github.com/tensorflow/swift/blob/master/Sources/SIL/SILPrinter.swift#L135-L138).